src/test/java/guitests/KeyboardShortcutsTest.java

```diff
@@ -171,6 +171,7 @@ public void keyboardShortcutsTest() {
         assertEquals(repoSelectorComboBox.getValue(), "dummy/dummy");
         // test shortcut when focus is on panel
         PlatformEx.waitOnFxThread();
+        sleep(1000);
```

Comment:
Is there another way to synchronise? How about waiting for col1 to be visible, or for an issue card to be selected?